liblangutil/EVMVersion.h

```diff
@@ -96,6 +96,7 @@ class EVMVersion:
 	bool hasChainID() const { return *this >= istanbul(); }
 	bool hasSelfBalance() const { return *this >= istanbul(); }
 	bool hasBaseFee() const { return *this >= london(); }
+	bool supportsPrevRandao() const { return *this >= paris(); }
```

Comment:
```suggestion
	bool hasPrevRandao() const { return *this >= paris(); }
```

Why deviate from the convention above? For any opcode, it's ``has`` - ``Returndata`` is not only an opcode, but several opcodes and changes to the call opcodes, so deviating there makes sense, but here it can just be ``has``.